lib/menu.rb

```diff
@@ -0,0 +1,15 @@
+require 'csv'
+
+class Menu < Array
+
+  def initialize
+    load_menu
+  end
+
+  def load_menu
+    CSV.foreach("menu.csv", headers: true, header_converters: :symbol, converters: :numeric) do |row|

```

Comment:
I like this way of loading the menu, as it implies that it can be done by someone non-technical and then loaded into the programme, which is probably how it would work in real life